example/lib/main.dart

```diff
@@ -6,14 +6,15 @@ void main() {
 }
 
 @Openapi(
-    additionalProperties:
-        DioProperties(pubName: 'petstore_api', pubAuthor: 'Johnny dep..'),
-    inputSpecFile: 'openapi-spec.yaml',
-    typeMappings: {'Pet': 'ExamplePet'},
-    generatorName: Generator.dio,
-    runSourceGenOnOutput: true,
-    alwaysRun: true,
-    outputDirectory: 'api/petstore_api')
+  additionalProperties:
+      DioProperties(pubName: 'petstore_api', pubAuthor: 'Johnny dep..'),
+  inputSpecFile: 'openapi-spec.yaml',
+  typeMappings: {'Pet': 'ExamplePet'},
+  generatorName: Generator.dio,
+  runSourceGenOnOutput: true,
+  alwaysRun: true,
+  outputDirectory: 'api/petstore_api',
+)
 class MyApp extends StatelessWidget {
```

Comment:
This looks like it may be due to the nature of how build_runner actually works. Currently build_runner doesn't run when dart assets don't change. I'm going to file an issue to see if there is a way to have it always run 🤔

Reply:
I've opened: https://github.com/dart-lang/build/issues/3561

Reply:
I found a gross workaround `rm .dart_tool/build/**/asset_graph.json && flutter|dart pub run build_runner build` which effectively invalidates the asset_graph and forces the generator to rebuild.

Reply:
But that would affect other libraries would it not? I do not think that is right.

Reply:
> This looks like it may be due to the nature of how build_runner actually works. Currently build_runner doesn't run when dart assets don't change. I'm going to file an issue to see if there is a way to have it always run 🤔

Indeed it is. It's fine either way. This would not block this PR.

Reply:
It absolutely would and I bring that up as a concern to within the issue linked above as this would cause all builders to need to be rerun which isn't preferable. But it does help target where the flag should be applied to provide better support.

Reply:
what If after running the generator, you add a comment to the class that has the ```openapi``` annotation. Something like ```///Last Processed: 31-12-2023 12:55:12pm```. That way, the file is modified after the build_runner has done its thing. Do you think that would work? Apply it only to next gen of course. If it's not feasible, it's fine.

Reply:
As a work around I think that is fine.